### PR TITLE
zh-trans: modify /docs/reference/glossary/endpoint-slice.md

### PR DESCRIPTION
**Pull request by lpf7551321**
zh-trans: modify /docs/reference/glossary/endpoint-slice.md
Update: /docs/reference/glossary/endpoint-slice.md
Co-Authored-By: lpf7551321

### CODE REVIEW

File: content/zh/docs/reference/glossary/endpoint-slice.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Endpoint 切片
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/endpoint-slice.md]: 端点切片

File: content/zh/docs/reference/glossary/endpoint-slice.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+<!--more-->
+
+<!--
+A scalable and extensible way to group network endpoints together. These can be
+used by {{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}} to
+establish network routes on each {{< glossary_tooltip text="node" term_id="node" >}}.
+-->
+
+
+一种将网络端点组合在一起的可扩缩、可扩展方式。它们将被 {{< glossary_tooltip text = "kube 代理" term_id = "kube-proxy" >}} 用于在每个 {{< glossary_tooltip text ="节点" term_id = "node">}} 上建立网络路由。
```
Comment (xichengliudui) [inline content/zh/docs/reference/glossary/endpoint-slice.md]: kube 代理 -> kube-proxy

**Comment by xichengliudui** [comment]
/lgtm
/hold cancel
@lpf7551321 thanks!

**Pull request merged**